invisible_cities/cities/esmeralda.py

```diff
@@ -170,6 +171,9 @@ def track_blob_info_creator_extractor(vox_size         : [float, float, float],
     A function that from a given HitCollection returns a pandas DataFrame with per track information.
     """
     def create_extract_track_blob_info(hitc):
+        df = pd.DataFrame(columns=list(types_dict_tracks.keys()))
+        if len(hitc.hits)>max_num_hits:
+            return df, hitc, True
```

Comment:
Is this `out_of_map = True` always correct? Does it have a meaning in this case at all?

Reply:
In practice it seems to be, in theory it doesnt have to.. Since the events will be saved in summary table the correct thing would be either to rename the column or add a new one (ie 'good_event'). However since most of the data has already been processed I think is best to leave it as is. I fixed it to True since most of the analysis using blobs and tracks properties will only be valid for events that are inside the active volume, so it will skip this weird large events.